deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -981,7 +981,7 @@ def deploy_build(request, name, stage, build_id):
 
 def deploy_commit(request, name, stage, commit):
     env = environs_helper.get_env_by_stage(request, name, stage)
-    builds = builds_helper.get_builds_and_tags(request, commit=commit)
+    builds = builds_helper.get_builds_and_tags(request, name=env.get('buildName', ''), commit=commit)
```

Comment:
Is this possible if env does not contain 'buildName'? Do we need handle the case when name = None?

Reply:
In my understanding it's possible but rare for an env to not have a 'buildName'. When it does not have a buildName, Teletraan will not display builds on the left for it. Here the implementation is when buildName does not exist, will search only by commit SHA.